dockerutil/kill.go

```diff
@@ -2,24 +2,36 @@ package dockerutil
 
 import (
 	"context"
+	"os"
 	"strings"
 
 	dockerapitypes "github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/filters"
 	"github.com/docker/docker/client"
 )
 
 // KillAllInterchaintestContainers kills all containers that are prefixed with interchaintest specific container names.
 // This can be called during cleanup, which is especially useful when running tests which fail to cleanup after themselves.
 // Specifically, on failed ic.Build(...) calls.
 func KillAllInterchaintestContainers(ctx context.Context) []string {
+	if os.Getenv("KEEP_CONTAINERS") != "" {
+		return nil
+	}
+
```

Comment:
Would it make sense to make this != "false"? Might also be worth adding this to the docs

Reply:
ICT has this other check where this is the standard
![image](https://github.com/user-attachments/assets/99feca3a-7db3-4050-bb8b-9c1c64cd360e)

evmOptions.md explains it
![image](https://github.com/user-attachments/assets/f78361ee-268e-44df-8bd0-9a455ca6a430)

Reply:
Oh perfect! Sounds good with me